Controllers/ProductsController.cs

```diff
@@ -44,6 +56,70 @@ public ProductsController(BangazonContext ctx)
             return View(product);
         }
 
+
+        public async Task<IActionResult> EditInfo([FromRoute]int? id)
+        {
+            ViewData["ProductTypeId"] = context.ProductType
+                .OrderBy(l => l.Label)
+                .AsEnumerable()
+                .Select(li => new SelectListItem { 
+                    Text = li.Label,
+                    Value = li.ProductTypeId.ToString()
+                    });
+                                        
+            // If no id was in the route, return 404
+            if (id == null)
+            {
+                return NotFound();
+            }
+
+            var product = await context.Product
+                    .Include(s => s.User)
+                    .SingleOrDefaultAsync(m => m.ProductId == id);
+
+            // If product not found, return 404
+            if (product == null)
+            {
+                return NotFound();
+            }
+
+            return View(product);
+        }
+
+        [ValidateAntiForgeryToken]
+        public async  Task <IActionResult> Edit([FromRoute]int id, Product product)
+        {
+            if (!ModelState.IsValid)
+            {
+                return View(product);
+            }
+            
+            if (id == null)
+            {
+                return BadRequest(product);
+            }
+
+            
+            Product originalProduct = context.Product.Single(p => p.ProductId == id);
+            originalProduct.ProductId = (int)id;
+            originalProduct.Description = product.Description;
+            originalProduct.Name = product.Name;
+            originalProduct.ProductTypeId = product.ProductTypeId;
+            context.Entry(originalProduct).State = EntityState.Modified;
+
+            context.Update(originalProduct);
+            try
+            {
+                context.SaveChanges();
+            }
+            catch (DbUpdateException)
+            {
+                throw;
+            }
+            
+            return RedirectToAction("Index", "Products");

```

Comment:
Let's send this to ProductTypes/Index to match up with my other PRs
`return RedirectToAction("Index", "ProductTypes");`